### DISCUSSION TITLE
Customize Dashboard

### DISCUSSION THREAD

**Discussion by TheRealFlyingCoder** [saleor/saleor#10736] [Q&A] [answered]
So my understanding is that there's no "Approved" method of customizing the look and feel of the Dashboard right?

I will be implementing a heavily customized storefront project soon, and the Dashboard order listing will need a different layout to what's currently there.

Is there an "Unapproved" method of customizing? I've seen mentions of forking Macaw?

**Comment by peelar** [comment, marked as answer]
Hello @TheRealFlyingCoder 👋 

I am afraid we have no way to affect the UI of the Dashboard without forking it. If you were to fork it, you become responsible for making sure it stays up to date with the original repository (if you want that). The same goes for our [Storefront](https://github.com/saleor/react-storefront).

**Reply by TheRealFlyingCoder** [reply]
All good, that's what I thought!

Were there any features like this in the pipeline?

This is a huge killer for us, maintaining a forked repo is a lot of overhead just to add extra columns into the order listing table.

I'm sure it's been brought up by other people before, customisation is key to a personalised and happy experience 😅

**Reply by peelar** [reply]
Currently, you either use the Dashboard as it is or fork it and accept the maintenance cost. We recognize a massive gap between those options, so I am sure we will try to decrease it in the future. However, I can't promise anything substantial in the next couple of months.

If you want to extend the functionalities of Dashboard, your best bet is to create an app within it. Here are some resources on it:
- https://docs.saleor.io/docs/3.x/developer/extending/apps/key-concepts
- https://learn.saleor.io/cli/creating-apps/
